src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/domene/FagsakRepository.kt

```diff
@@ -49,4 +49,25 @@ interface FagsakRepository : JpaRepository<Fagsak, Long> {
         nativeQuery = true,
     )
     fun finnFagsakerSomSkalAvsluttes(): List<Fagsak>
+
+    @Query(
+        value = """
+                    WITH sisteiverksatte AS (SELECT DISTINCT ON (b.fk_fagsak_id) b.id, b.fk_fagsak_id, b.opprettet_tid
+                                 FROM behandling b
+                                          INNER JOIN tilkjent_ytelse ty ON b.id = ty.fk_behandling_id
+                                 WHERE b.status = 'AVSLUTTET'
+                                   AND ty.utbetalingsoppdrag IS NOT NULL
+                                 ORDER BY b.fk_fagsak_id, b.aktivert_tid DESC)
+        
+        SELECT f.*
+        FROM fagsak f
+                 JOIN behandling b ON f.id = b.fk_fagsak_id
+                 JOIN sisteiverksatte sistiverksattBehandling ON sistiverksattBehandling.id = b.id
+                 JOIN andel_tilkjent_ytelse aty ON aty.fk_behandling_id = b.id
+        WHERE aty.stonad_tom > '2024-07-31 00:00:00'
+          AND aty.stonad_tom < '2025-01-01 00:00:00'
+        """,
+        nativeQuery = true,
+    )
+    fun finnFagsakerSomHarSistIverksattBehandlingMedUtbetalingSomStopperMellomAugOgDes2024(): List<Fagsak>
```

Comment:
Man bruker kun fagsakId videre, så denne kan returnere en liste med Long.  Lista er nok ikke så stor, men da sparer man både minne og alle joinsa som hibernate må gjøre mot Aktør. For store lister er man i fare for å få OOM